### ISSUE TITLE
Find the guy who's maintaining the choco package

### ISSUE THREAD

**Issue by YoshiRulz** [TASVideos/BizHawk#2655]
~~and whap 'em 🗞️~~ and ask them politely to give us whatever keys/perms you need for making changes

**Comment by zeromus** [comment]
no. then it's our problem.
rewrite this issue as "make conscious decision to adopt choco distribution" so it's clear this is increased scope.
and then as a subtask "for better quality, adopt the existing choco package"

**Comment by YoshiRulz** [comment]
This is primarily about *security*, and secondarily about having the choco package be useful.

**Comment by zeromus** [comment]
You can't control everyone distributing your shit in weird packages. Don't even start trying.

**Comment by Meerkov** [comment]
I agree with Zeromus here. There are dozens of package distribution systems. It's not scalable to try to own and manage your package in all of them, and I'm not sure your MIT license allows you to try to _prevent_ other people from distributing it either.

> Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software

The only argument here then, is whether you don't want them to re-use the name "BizHawk" because you think it falsely implies that it's an official distribution. In that case, you could presumably reach out to the site admins and ask they give you the "bizhawk" package name. But I'm not sure it's a great use of time to do the same for every other package distribution system either.

**Comment by YoshiRulz** [comment]
If our binaries were signed I wouldn't care nearly as much. Your point about the license is correct though.

**Comment by YoshiRulz** [comment]
@Xav83 Hey, thanks for putting in the work for the choco package. Could you please push 2.10?
